bin/setup

```diff
@@ -18,3 +18,10 @@ fi
 mix ecto.create
 npm install yarn
 ./node_modules/.bin/yarn install
+
+if [ "$CI" ]; then
```

Comment:
Took it from https://semaphoreci.com/community/tutorials/end-to-end-testing-in-elixir-with-hound